### DISCUSSION TITLE
server.secretkey is missing when installing from core-install

### DISCUSSION THREAD

**Discussion by rafalkrupinski** [argoproj/argo-cd#12835] [Q&A] [answered]
Hi,
It's my first installation of Argo CD.

I'm installing from core (https://argo-cd.readthedocs.io/en/stable/operator-manual/installation/#core) on k3s but I can't install any apps.
logs from application controller show:
> time="2023-03-13T10:15:38Z" level=warning msg="Failed to save clusters info: server.secretkey is missing"
> time="2023-03-13T10:15:38Z" level=error msg="Partial success when performing preferred resource discovery" error="unable to retrieve the complete list of server APIs: metrics.k8s.io/v1beta1: the server is currently unable to handle the request, traefik.containo.us/v1alpha1: the server could not find the requested resource" server="https://kubernetes.default.svc"
> time="2023-03-13T10:15:38Z" level=warning msg="failed to set app resource tree: dial tcp 10.43.145.167:6379: connect: connection refused"

I've searched the missing secretkey, but solutions assume the argocd server is running but that's not the case with core-install.
My logs are flooded with that missing secretkey message.

I'll try the normal installation but it would be great if I could use core.

**Comment by crenshaw-dev** [comment, marked as answer]
I believe this is a bug. I've created an issue to track it: https://github.com/argoproj/argo-cd/issues/12903
